### ISSUE TITLE
How to export OBJ files instead of the video?

### ISSUE THREAD

**Issue by hungdche** [magic-research/GETAvatar#3]
Thanks for such an awesome work! How could I export the obj files of the human meshes?

**Comment by Xuanmeng-Zhang** [comment]
Yes, I think we can extract the textureless mesh with marching cude function.

**Comment by hungdche** [comment]
Thank you. One more question, the provided command for inference requires the dataset. Can I infer from the model without providing the dataset?

**Comment by Xuanmeng-Zhang** [comment]
Yes, during inference, you can define your own SMPL parameters to generate the corresponding human.

**Comment by hungdche** [comment]
If you could guide me on how to do that or give a few pointers I would greatly appreciate it!

**Comment by Xuanmeng-Zhang** [comment]
We use the dataset to load the labels, so you can pass the steps and directly define the labels at [here](https://github.com/magic-research/GETAvatar/blob/943d43c49f789b24a71a693bdc4b44e9d9c08272/training/multi_view_3d.py#L88).

**Comment by hungdche** [comment]
Hi @Xuanmeng-Zhang, sorry for reopening this. The only animation I want to use is walking, so I think of getting the walking SMPL from [here](https://replicate.com/daanelson/motion_diffusion_model). But I have no idea how to convert that SMPL to the one defined [here](https://github.com/magic-research/GETAvatar/blob/943d43c49f789b24a71a693bdc4b44e9d9c08272/training/dataset.py#L259). My ultimate goal is to generate a 3D human mesh with walking animation. I hope you can help me out a bit. Thank you so much!

**Comment by mayank64ce** [comment]
@hungdche I think I did managed to do that. If I remember correctly, there was some way to render the output of `motion diffusion model` using `human_body_prior` library or soemthing. It gave out a `.npz` file. I got the `72` parametres from each frame and put them in the `smpl_array` attribute just like the `mocap` data GETAvatar uses. and saved it in .pkl format. 

Then you can load it like a regular animation sequence as mentioned in their readme. Although I could not stop the camera from rotating around the subject, it did perform the motion.
